sdk/inc/azure/core/_az_cfg_prefix.h

```diff
@@ -2,9 +2,10 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @file _az_cfg_prefix.h
+ * @file
  *
- * @brief Open "extern C" and save warnings state.
+ * @brief Opens "extern C" and saves warnings state. Do not include this file directly. This file
```

Comment:
This is repetitive.  The same statement is made is the note about using '_' items.

Reply:
But on the other hand, the statement does not explicitly mention files I'd double-mention in this case, but I don't have strong opinion, of others think I should remove this note.